### PR TITLE
React static list of todos

### PR DESCRIPTION
**Pull request by vsupruniuk**
[DEMO LINK](https://vsupruniuk.github.io/react_static-list-of-todos/)

### CODE REVIEW

File: src/components/TodoInfo/TodoInfo.tsx [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
 
-// Create a `TodoInfo` component accepting a `todo` object and use it in the
-// list to render `title`, `completed` status and `User`
+import './TodoInfo.scss';
 
-// Add a default export statement for TodoInfo component to use it in the other files
+type Props = {
+  title: string,
+  completed: boolean,
+  userId: number,
```
Comment (AndriiNychaiuk) [inline src/components/TodoInfo/TodoInfo.tsx:8]: Same here 😅. You should use your interface instead of all these parameters

**Review by AndriiNychaiuk — APPROVED** [review]

File: src/components/TodoInfo/TodoInfo.tsx (hunk 1 above)
Comment (vsupruniuk) [reply src/components/TodoInfo/TodoInfo.tsx:8]: In this case, I need to pass the `user` parameter to the `TodoInfo` component. I think it's not a good idea to pass a parameter that won't be used

File: src/components/TodoInfo/TodoInfo.tsx [hunk 2]
```diff
@@ -1,6 +1,19 @@
-// Don't forget to import the React library
+import React from 'react';
 
-// Create a `TodoInfo` component accepting a `todo` object and use it in the
-// list to render `title`, `completed` status and `User`
+import './TodoInfo.scss';
 
-// Add a default export statement for TodoInfo component to use it in the other files
+type Props = {
```
Comment (jstmpelowycz) [inline src/components/TodoInfo/TodoInfo.tsx:5]:
```suggestion
type Props = Todo;
```
simplify your life :)

File: src/components/TodoInfo/TodoInfo.tsx [hunk 3]
```diff
@@ -1,6 +1,19 @@
...
+  title: string,
+  completed: boolean,
+  userId: number,
+};
+
+const TodoInfo: React.FC<Props> = ({ title, completed, userId }) => (
+  <>
+    <div className="todo-info">{`Task: ${title}`}</div>
+    <div className="todo-info" data-cy="status">{`Status: ${completed ? 'Done' : 'In progress'}`}</div>
+    <div className="todo-info">{`User: ${userId}`}</div>
```
Comment (jstmpelowycz) [inline src/components/TodoInfo/TodoInfo.tsx:15]: formatting

**Review by jstmpelowycz — CHANGES_REQUESTED** [review]

File: src/components/TodoInfo/TodoInfo.tsx (hunk 1 above)
Comment (AndriiNychaiuk) [reply src/components/TodoInfo/TodoInfo.tsx:8]: It doesn't work like this, does it? You'll choose what exactly you'll pass to the todoInfo. Like todo.title, todo.userId, etc

File: src/components/TodoList/TodoList.tsx [hunk 4]
```diff
@@ -1,6 +1,27 @@
...
 
-// Add a default export statement for TodoInfo component to use it in the other files
+type Props = {
+  todos: Todo[]
+};
+
+const TodoList: React.FC<Props> = ({ todos }) => (
+  <ul className="list">
+    {todos.map(todo => (
+      <li className="list__item">
```
Comment (jstmpelowycz) [inline src/components/TodoList/TodoList.tsx:15]: key?

**Review by jstmpelowycz — APPROVED** [review]

**Pull request closed without merging**